source/_integrations/samsungtv.markdown

```diff
@@ -54,9 +54,8 @@ After saving the YAML configuration, the TV must be turned on _before_ launching
 
 ### Turn on action
 
-If the integration knows the MAC address of the TV from discovery, it will attempt to wake it using wake on LAN when calling turn on. Wake on LAN must be enabled on the TV for this to work. If the TV is connected to a smart strip or requires a more complex turn-on process, a `turn_on` action can be provided that will take precedence over the built-in wake on LAN functionality.
+If Wake on LAN is enabled on the TV, or if the TV is connected to a smart strip or requires a more complex turn-on process, a `turn_on` automation can be created from the user interface. From the device create a new automation and select the **Device is requested to turn on** automation.
```

Comment:
How should the user find the correct mac address?

Reply:
In the core PR, I have added the MAC address to the issue description, so it is visible straight away (maybe we keep the issue even after deprecation...)

For newer models, it should also be available in the integration diagnostics (I added a note here).
For older models I have no idea... maybe via the router?